docker/oap/Dockerfile.oap

```diff
@@ -14,28 +14,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG BASE_IMAGE='adoptopenjdk/openjdk11:alpine'
+ARG BASE_IMAGE='adoptopenjdk/openjdk11:alpine-jre'
```

Comment:
We need to update https://github.com/apache/skywalking/blob/edcf5926a84f4cc31d702f7014b9a6b5769d8b7a/.github/workflows/e2e.jdk-versions.yaml#L40 so that different versions of `jre` can be tested.

Reply:
Gotcha. I also found some other `openjdk` in e2e tests, for example, istio relevant cases.

Reply:
Do we have jre agent image already?

Reply:
> Do we have jre agent image already?

For Java agent, no, it's still using JDK

Reply:
OK, seems we should take care of it later.